### DISCUSSION TITLE
Winget command via remote background

### DISCUSSION THREAD

**Discussion by djbotelho** [amidaware/tacticalrmm#1690] [General]
![image](https://github.com/amidaware/tacticalrmm/assets/113935790/e9711c97-f606-4ac0-bc21-58e40120e20e)

Cant use the winget command to try and uninstall software remotely, any suggestion?

**Comment by dinger1986** [comment]
winget isnt accessible in the system context, we have a community script which can be used via tactical scripts this finds the location and then goes to it and then allows winget to be ran.
```
$wingetloc=(Get-Childitem -Path "C:\Program Files\WindowsApps" -Include winget.exe -Recurse -ErrorAction SilentlyContinue | Select-Object -Last 1 | %{$_.FullName} | Split-Path)
cd $wingetloc
```

you can use that in remote background with powershell or use the community scripts provided which should be alot easier :)
